common/src/main/java/com/redhat/cloud/notifications/models/WebhookProperties.java

```diff
@@ -32,10 +35,25 @@ public class WebhookProperties extends EndpointProperties {
     @JsonProperty("secret_token")
     private String secretToken; // TODO Should be optional
 
+    /**
+     * The ID of the "secret token" secret in the Sources backend.
+     */
+    @Column(name = "secret_token_id")
+    @JsonIgnore
+    private long secretTokenSourcesId;
```

Comment:
Lets use the Object-type equivalent as we could have `null` values in the database - unless the migrations sets a default value.

```suggestion
    private Long secretTokenSourcesId;
```

Reply:
Yeah, makes sense!